core/domain/user_services.py

```diff
@@ -2220,10 +2506,17 @@ def get_contributor_usernames(category, language_code=None):
     else:
         raise Exception('Invalid category: %s' % category)
 
-    return get_usernames(user_ids)
+    usernames = get_usernames(user_ids, strict=True)
+    # Ruling out the possibility of Sequence for mypy type checking.
+    assert isinstance(usernames, list)
+    return usernames
```

Comment:
I think it should be fine to return sequence from this function.

Reply:
Done!